src/schedule.jl

```diff
@@ -146,12 +146,33 @@ function _print_running_talks(current_talks; now=default_now())
     return nothing
 end
 
-function now(; now=default_now())
+function now(now, ::Val{:text})
+    current_talks = get_running_talks(; now=now)
+    str = ""
+    if !isnothing(current_talks)
+        for (track, talk) in current_talks
+            str *= """
+            $track
+            \t$(talk.title) ($(talk.type))
+            \t├─ $(JuliaCon.speakers2str(talk.speaker))
+            \t└─ $(talk.url)
+            """
+        end
+    end
+    str *= "\n(Full schedule: https://pretalx.com/juliacon2021/schedule)"
+    return str
+end
+
+function now(now, ::Val{:terminal})
     current_talks = get_running_talks(; now=now)
     _print_running_talks(current_talks; now=now)
     return nothing
 end
 
+# A dispatcher for the `now` methods. Default to terminal output.
+now(; now=default_now(), output=:terminal) = JuliaCon.now(now, Val(output))
```

Comment:
I think we should keep `now` as a keyword argument, i.e.

```Julia
now(; output=:terminal, now=default_now()) = JuliaCon.now(Val(output); now)
```

and then

```Julia
function now(::Val{:text}; now)
 ...
```

etc. (same for `today()`).

Reply:
That the part I prefer of using a dispatcher! Not that it matters much anyway ;)

OK for keeping it as a keyword argument